src/main/java/uk/gov/hmcts/cmc/claimstore/config/properties/notifications/NotificationsProperties.java

```diff
@@ -20,6 +20,10 @@
     @NotEmpty
     private String frontendBaseUrl;
 
+    @URL
+    @NotEmpty
+    private String respondToClaimUrl;
```

Comment:
also need a PR to integration-tests repo to add this for docker environment, that can be merged before this